Source/diablo.cpp

```diff
@@ -377,6 +377,10 @@ BOOL StartGame(BOOL bNewGame, BOOL bSinglePlayer)
 			break;
 		}
 
+		// Save 1.3 MiB of RAM by freeing all main menu resources
+		// before starting the game.

```

Comment:
Do you think the comments are really needed here? Isn't the best practice of "releasing as soon as possible" enough justification to do it?

Reply:
I'd like to keep the comment because the control flow is very subtle here.